testdata/picard/fingerprint/NA12891andNA12892.vcf

```diff
@@ -0,0 +1,101 @@
+##INFO=<ID=AC,Number=A,Type=Integer,Description="Allele count in genotypes, for each ALT allele, in the same order as listed">
+##INFO=<ID=AF,Number=A,Type=Float,Description="Allele Frequency, for each ALT allele, in the same order as listed">
+##INFO=<ID=AN,Number=1,Type=Integer,Description="Total number of alleles in called genotypes">
+##INFO=<ID=DP,Number=1,Type=Integer,Description="Approximate read depth; some reads may have been filtered">
+##contig=<ID=1,length=249250621>
```

Comment:
Do you need all of the ##contigs since it looks like you are only testing variants on 1, 3, and 7?

Reply:
yes.

the program compares dictionaries and wants an exact match....I guess I could edit all the input files and shrink their dictionaries....is it worth it?